parlai/tasks/task_list.py

```diff
@@ -1130,4 +1130,14 @@
             "<https://arxiv.org/abs/1911.03842>."
         ),
     },
+    {
+        "id": "ReDial",
+        "display_name": "ReDial",
+        "task": "redial",
+        "tags": ["All", "ChitChat"],
```

Comment:
we could also add the `Goal` tag since you could see movie recommendation as task/goal-oriented